tests/wpt/tests/fetch/range/cache-revalidate-construct-range-response.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<meta name="timeout" content="long">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<body>
+<script>
+
+promise_test(async t => {
+    const url = new URL('resources/video-with-aged-proxy-cache.py', location.href);
+    const response_1 = await fetch(url.toString());
```

Comment:
In my previous review I had a nit for this. Make sure to address review comments or acknowledge them if you disagree with them. This is a nit, so we can merge this now, but in the future please do ensure you address the full review.